src/gui/properties/trackerlistwidget.cpp

```diff
@@ -150,9 +150,9 @@ QList<QTreeWidgetItem*> TrackerListWidget::getSelectedTrackerItems() const
 
 void TrackerListWidget::setRowColor(int row, QColor color)
 {
-    unsigned int nbColumns = columnCount();
+    int nbColumns = columnCount();
```

Comment:
Since you're at it, how about adding also `const` to it whenever possible?

Reply:
Certainly.